src/backer/cardaning.py

```diff
@@ -72,43 +70,60 @@ def updateTip(self, tipHeight):
         self.tipHeight = tipHeight
 
     def publishEvent(self, event: bytes):
-        self.pending_kel = event + self.pending_kel
+        self.pending_kel.insert(0, event)
```

Comment:
I thought this stored it to the DB. It's been a while since I've looked at that part of the code. Is this not the case?

Reply:
Yep, I keep this variable from the old code, I also want to ask you to change this => we need to read/remove directly from the DB (kel_queued) for safe so we don't use self.pending_kel anymore

Reply:
Yeah, when we receive a KEL via the API to receipt, we should store in the DB instead of in memory. Can you make this change?

Reply:
Yes, I will do that